### PR TITLE
[api] handle lost projects in backend in consistency check/fix

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

**Comment by adrianschroeter** [comment]
went in by accident together with a backend fix...

keep this open for review purposes, but feel free to close.

File: src/api/app/jobs/consistency_check.rb [hunk 1]
```diff
@@ -58,6 +59,15 @@ def check_project(fix = nil)
       @errors << import_project_from_backend(ENV['project'])
       project = Project.get_by_name(ENV['project'])
     end
+    # check backend side
+    begin
+      Suse::Backend.get("/source/#{project.name}")
+    rescue ActiveXML::Transport::NotFoundError
+      Suse::Backend.without_global_write_through do
```
Comment (hennevogel) [inline src/api/app/jobs/consistency_check.rb]: There is also a `:no_backend_write` option in `commit_opts` for Project. I think that is easier to understand...

Comment (adrianschroeter) [inline src/api/app/jobs/consistency_check.rb]: okay, done in all places in request https://github.com/openSUSE/open-build-service/pull/2901

**Comment by adrianschroeter** [comment]
obsoleted by https://github.com/openSUSE/open-build-service/pull/2901

**Pull request closed without merging**